src/main/java/gov/usgs/cida/pubs/dao/ipds/IpdsMessageLogDao.java

```diff
@@ -26,50 +22,40 @@ public IpdsMessageLogDao(SqlSessionFactory sqlSessionFactory) {
 
 	private static final String NS = "ipdsMessageLog";
 
-	/** {@inheritDoc}
-	 * @see gov.usgs.cida.mypubsJMS.dao.intfc.IDao#add(java.lang.Object)
-	 */
 	@Transactional
 	@Override
 	public Integer add(IpdsMessageLog domainObject) {
 		return insert(NS + ADD, domainObject);
 	}
 
-	/** {@inheritDoc}
-	 * @see gov.usgs.cida.mypubsJMS.dao.intfc.IDao#getById(java.lang.Integer)
-	 */
 	@Transactional(readOnly = true)
 	@Override
 	public IpdsMessageLog getById(Integer domainID) {
 		return (IpdsMessageLog) getSqlSession().selectOne(NS + GET_BY_ID, domainID);
 	}
 
-	/** 
-	 * {@inheritDoc}
-	 * @see gov.usgs.cida.pubs.core.dao.BaseDao#getByMap(Map)
-	 */
 	@Transactional(readOnly = true)
 	@Override
 	public List<IpdsMessageLog> getByMap(Map<String, Object> filters) {
 		return getSqlSession().selectList(NS + GET_BY_MAP);
 	}
 
-	/** {@inheritDoc}
-	 * @see gov.usgs.cida.pubs.core.dao.intfc.IDao#update(java.lang.Object)
-	 */
 	@Transactional
 	@Override
 	public void update(IpdsMessageLog domainObject) {
 		update(NS + UPDATE, domainObject);
 	}
 
-	/** {@inheritDoc}
-	 * @see gov.usgs.cida.dao.intfc.IIpdsMessageLogDao#getFromIpds(java.lang.Integer)
-	 */
 	@Transactional(readOnly = true)
 	@Override
 	public List<Map<String, Object>> getFromIpds(Integer ipdsMessageLogId) {
 		return getSqlSession().selectList(NS + ".getMpPublicationFromIpds", ipdsMessageLogId);
 	}
 
+	@Transactional(readOnly = true)
+	@Override
+	public List<Map<String, Object>> getFromSipp(Integer ipdsMessageLogId) {
```

Comment:
Seems a little strange that sipp messages are part of of the IPDSMessageLogDao?

Reply:
This will be cleaned up in the next JIRA task.